database/schema/upgrade_scripts/122-vulnerable_package_tables.sql

```diff
@@ -0,0 +1,150 @@
+-- package name table
+CREATE TABLE package_name (
+  id BIGSERIAL,
+  name TEXT NOT NULL UNIQUE, CHECK (NOT empty(name)),
+  PRIMARY KEY (id)
+) TABLESPACE pg_default;
+
+GRANT SELECT, INSERT, UPDATE, DELETE ON package_name TO ve_db_user_evaluator;
```

Comment:
Let's allow modifying operations (INSERT, UPDATE, DELETE) only for the ve_db_user_evaluator user for now. Alow selects for all other users.